packages/hd-transport/scripts/protobuf-build.sh

```
@@ -29,24 +30,46 @@ if [[ "$LANG" != "typescript" && "$LANG" != "flow" ]];
-npx pbjs -t json -p $DIST -o $DIST/messages.json --keep-case messages.proto
-rm $DIST/messages.proto
+if [ -f "$MESSAGES_JSON_PATH" ]; then
+    echo "Using existing messages.json from onekey-protocol submodule"
+    cp "$MESSAGES_JSON_PATH" "$DIST/messages.json"
+else
+    echo "No existing messages.json found in onekey-protocol, generating from .proto files"
+    
+    # BUILD combined messages.proto file from protobuf files
+    # this code was copied from ./submodules/onekey-protocol/protob Makekile
+    # clear protobuf syntax and remove unknown values to be able to work with proto2js
+    echo 'syntax = "proto2";' > $DIST/messages.proto
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)